stdlib/toolchain/Compatibility51/Overrides.h

```diff
@@ -0,0 +1,33 @@
+//===--- Overrides.h - Compat overrides for Swift 5.0 runtime ------s------===//
```

Comment:
Should this file be called `ProtocolConformance.h` instead?

Reply:
If we add more hooks in the future, my idea is that we'd declare them all in the header here, but possibly fan out the implementations into separate .cpp files.

Reply:
OK, makes sense to me!